include/kodlab_mjbots_sdk/pid_controller.h

```diff
@@ -0,0 +1,648 @@
+/**
+ * @file pid.h
+ * @author Chandravaran Kunjeti (kunjeti@seas.upenn.edu)
+ * @author Raghavesh Viswanath (vrag@seas.upenn.edu)
+ * @brief Class implementation of PID
+ * @date 5/18/2023
+ * @copyright 2023 The Trustees of the University of Pennsylvania.
+ * All rights reserved.
+ *
+ */
+
+#pragma once
+
+#include <math.h>
+#include <array>
+#include <vector>
+#include <algorithm>
+#include <Eigen/Dense>
+#include <tuple>
+
+namespace kodlab {
+
+/**
+ * @brief PIDController class that allows you to use PID control.
+ */
+
+template <typename Scalar, int N=1>
```

Comment:
Probably should add comments on what scalar and N are.

Reply:
Added